README.md

```diff
@@ -324,20 +325,20 @@ this is ensured by `filtermail` proxy.
 
 Postfix is configured to require valid TLS
 by setting [`smtp_tls_security_level`](https://www.postfix.org/postconf.5.html#smtp_tls_security_level) to `verify`.
-If emails don't arrive from a chatmail server to your server,
-the problem is likely that your server does not have a valid TLS certificate.
+If emails don't arrive at your chatmail relay server, 
+the problem is likely that your relay does not have a valid TLS certificate.
 
-You can test it by resolving `MX` records of your server domain
-and then connecting to MX servers (e.g `mx.example.org`) with
+You can test it by resolving `MX` records of your relay domain
+and then connecting to MX relays (e.g `mx.example.org`) with
 `openssl s_client -connect mx.example.org:25 -verify_hostname mx.example.org -verify_return_error -starttls smtp`
 from the host that has open port 25 to verify that certificate is valid.
 
-When providing a TLS certificate to your server,
+When providing a TLS certificate to your chatmail server, 
```

Comment:
```suggestion
When providing a TLS certificate to your chatmail relay, 
```

or "chatmail relay server" maybe? We could also specifically talk about nginx, postfix + dovecot here, but that would be overkill I think.